### PR TITLE
Feature/loki operator

### PR DESCRIPTION
**Pull request by computate**
This is to add loki-operator as an app to the project. 
Documentation for this task can be found in many links here:

  https://loki-operator.dev/docs/forwarding_logs_to_gateway.md/
  https://grafana.com/docs/loki/latest/api/#push-log-entries-to-loki

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 **/overlays/*-local/
```
Comment (dystewart) [inline .gitignore]: Remove this file from commit?

**Review by dystewart — CHANGES_REQUESTED** [review]

File: loki/base/clusterrolebindings/lokistack-gateway-auth-delegator.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+kind: ClusterRoleBinding
+metadata:
+  name: lokistack-gateway-auth-delegator
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:auth-delegator
+subjects:
+  - kind: ServiceAccount
+    name: lokistack-gateway
```
Comment (naved001) [inline loki/base/clusterrolebindings/lokistack-gateway-auth-delegator.yaml:11]: Where is this ServiceAccount created?

File: loki/base/clusterrolebindings/lokistack-tenant-logs.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+kind: ClusterRoleBinding
+metadata:
+  name: lokistack-tenant-logs
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: lokistack-tenant-logs
+subjects:
+- kind: ServiceAccount
+  name: logcollector
```
Comment (naved001) [inline loki/base/clusterrolebindings/lokistack-tenant-logs.yaml:11]: same; how's this created?

**Review by dystewart — APPROVED** [review]

**Comment by joachimweyl** [comment]
@dystewart was your CR resolved to your expectation?

File: loki/base/clusterrolebindings/lokistack-gateway-auth-delegator.yaml (hunk 2 above)
Comment (naved001) [reply loki/base/clusterrolebindings/lokistack-gateway-auth-delegator.yaml:11]: @computate Just wanted to make sure you saw my questions on this PR before this was merged. Perhaps, I should have used the "request changes" button.

**Pull request merged**